src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -124,6 +125,11 @@ const defaultProps = {
  * @param {Object} item
  */
 function dismissError(item) {
+    // Users can't dismiss company card errors because there's nothing they can do about it besides contacting their admin
+    if (item.canDismissError === false) {
+        return;
+    }
```

Comment:
NAB. I don't think this is necessary. The `dismissError` function can only be called though the `OfflineWithFeedback`'s `onClose` prop. That prop is already checked against `canDismissError`. IOW `dismissError` will never be called with `canDismissError` being `false`.

Reply:
Updated!